tests/gradients/core/test_hadamard_gradient.py

```diff
@@ -165,7 +174,7 @@ def test_pauli_rotation_gradient(self, G, theta, tol):
 
         assert len(tapes) == 1
 
-        assert isinstance(res_hadamard, np.ndarray)
+        # assert isinstance(res_hadamard, np.ndarray)
```

Comment:
Is this meant to be removed?

Reply:
It's now a float.  Which is equally fine.  I should probably just remove these lines entirely.

Reply:
```suggestion
```